db/schema.rb

```diff
@@ -20,11 +20,18 @@
   enable_extension "uuid-ossp"
 
   create_table "kudos", id: :uuid, default: -> { "uuid_generate_v4()" }, force: :cascade do |t|
-    t.text     "body",        null: false
+    t.text     "body",       null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.uuid     "giver_id",   null: false
```

Comment:
As Kudos usage increases, I think we'll want indexes on `giver_id` and `created_at`.
(Maybe not in this PR, but eventually!)

Reply:
I agree, the `created_at` index should be especially useful with paginated scrolling later.